components/ui/map/popup/LayerPopup.js

```diff
@@ -140,17 +141,27 @@ class LayerPopup extends React.Component {
           {(interaction.data || interactionState.data) &&
             <table className="popup-table">
               <tbody>
-                {interactionConfig.output.map(outputItem => (
-                  <tr
-                    className="dc"
-                    key={outputItem.property || outputItem.column}
-                  >
-                    <td className="dt">
-                      {outputItem.property || outputItem.column}:
-                    </td>
-                    <td className="dd">{this.formatValue(outputItem, (interaction.data || interactionState.data)[outputItem.column])}</td>
-                  </tr>
-                ))}
+                {output.map((outputItem) => {
+                  const { column } = outputItem;
```

Comment:
What about this. I think that using this function will prevent you to make a difference between normal ones and splitted ones
```
const columnArr = column.split('.');
const value = columnArray.reduce((acc, c) => {
  return acc[c];
}, interaction.data || interactionState.data)
```